packages/pyright-internal/src/common/serviceProviderExtensions.ts

```diff
@@ -77,6 +77,10 @@ ServiceProvider.prototype.console = function () {
     return this.get(ServiceKeys.console);
 };
 ServiceProvider.prototype.partialStubs = function () {
+    const result = this.tryGet(ServiceKeys.partialStubs);
+    if (!result) {
+        this.add(ServiceKeys.partialStubs, new PartialStubService(this.fs()));
```

Comment:
Partial stub support used to be part of the file system, so it shouldn't be optional.